ballerina-base-image/README.md

```diff
@@ -68,15 +66,7 @@ FROM ballerinalang/ballerina
 COPY files/* /ballerina/files/
 ```
 
-Notice that the only task the child Dockerfile does is to copy Ballerina packages to the standard location inside the image which is `/ballerina/files`. In this case, the Ballerina package will have to be a Ballerina `main`. For an image that packs a Ballerina Service archive, the Dockerfile would change to something like the following.
-
-```Dockerfile
-FROM ballerinalang/ballerina
-
-ENV SVC_MODE=true
-
-COPY files/* /ballerina/files/
-```
+Notice that the only task the child Dockerfile does is to copy Ballerina packages to the standard location inside the image which is `/ballerina/files`.
```

Comment:
It might still be better to put a Dockerfile example with the single COPY instruction IMO.

Reply:
Yes one sample is already there with single COPY instruction :). I have removed only service sample with,
ENV SVC_MODE=true
Removed since those environmental parameters no longer needed.

Reply:
Ack.